.github/scripts/set_benchmark_param.sh

```diff
@@ -15,9 +15,11 @@ fi
 if [[ $1 == *"llama2"* ]] || [[ $1 == *"Llama-2"* ]]
 then
     echo "MAX_ENTRY_COUNT=--cache-max-entry-count 0.95" >> "$GITHUB_ENV"
-
-else
+elif [[ $1 == *"internlm2-chat-20b"* ]]
```

Comment:
only llama and internlm2 should set --cache-max-entry-count value. the other models use default value